src/api/comments.ts

```diff
@@ -0,0 +1,14 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const createComment = (comment: Comment) => {
+  return client.post<Comment>('/comments', comment);
```

Comment:
It's better to pass only the required fields for creating a comment instead of the whole Comment object. Consider changing the function parameter to an object containing only the necessary fields.